bpf/Makefile

```diff
@@ -54,11 +54,11 @@ MAX_BASE_OPTIONS += -DENABLE_MASQUERADE_IPV4=1 -DENABLE_MASQUERADE_IPV6=1 \
 	-DENABLE_NODEPORT_ACCELERATION=1 -DENABLE_SESSION_AFFINITY=1 \
 	-DENABLE_DSR_ICMP_ERRORS=1 -DENABLE_DSR=1 -DENABLE_DSR_HYBRID=1 \
 	-DENABLE_IPV4_FRAGMENTS=1
-ifneq (,$(filter $(KERNEL),54 510 netnext))
+ifneq (,$(filter $(KERNEL),54 510 61 netnext))
```

Comment:
Only had this idea now: does it make sense to invert the condition? Basically only disable on old kernels instead of enabling on new ones. We'll keep adding new kernels but should only remove old ones.

Reply:
Yep I had the same thought, but wanted to keep the backport as simple & non-controversial as possible :).

Let's do this as follow-on for `main` only?

Reply:
https://github.com/cilium/cilium/pull/30127